src/Cache/FileCacheManager.php

```diff
@@ -77,6 +77,26 @@ public function __destruct()
         $this->writeCache();
     }
 
+    /**
```

Comment:
Maybe extract a `ForbiddSerialization` trait, to get the aspect into a single code-unit?

Reply:
I'm not sure it worth it...

Reply:
i'm good with both. I gonna merge as-is and if one is eager to create such trait, please raise a follow-up PR